src/server/location/location.service.ts

```diff
@@ -140,4 +145,25 @@ export class LocationService {
         .filter((title) => !!title),
     }));
   }
+
+  /**
+   * Resolves with a list of rooms along with their associated campus and
+   * building information. This is used to populate the Room Admin table.
```

Comment:
I would maybe remove this sentence about the room admin table - technically service methods shouldn't know (or care) about the front-end UI.